test/opsem/gint.ll

```diff
@@ -0,0 +1,77 @@
+; RUN: %seabmc "%s" 2>&1 | %oc %s
+; CHECK: ^unsat$
+; XFAIL: *
```

Comment:
This test initializes an `int` (no shadow instruction is inserted). The failure message is
```
Failed to marshal: store(select(main@%sm3, @yyyyyyy)_0, 0x8048004:bv(32), 1:bv(32))
#0 0x000000000213eeb4 PrintStackTraceSignalHandler(void*) (./run/bin/seahorn+0x213eeb4)
#1 0x000000000213f196 SignalHandler(int) (./run/bin/seahorn+0x213f196)
#2 0x00007fa3e71f2890 __restore_rt (/lib/x86_64-linux-gnu/libpthread.so.0+0x12890)
#3 0x000000000108f7e0 get_sort(expr const*) (./run/bin/seahorn+0x108f7e0)
#4 0x0000000000f3924c Z3_mk_store (./run/bin/seahorn+0xf3924c)
#5 0x0000000000d4f3e1 z3::ast ufo::BasicExprMarshal<ufo::FailMarshal>::marshal<boost::bimaps::views::unordered_map_view<boost::bimaps::relation::member_at::left, boost::bimaps::detail::bimap_core<boost::bimaps::unordered_set_of<boost::intrusive_ptr<expr::ENode>, boost::hash<boost::intrusive_ptr<expr::ENode> >, std::equal_to<boost::intrusive_ptr<expr::ENode> > >, boost::bimaps::unordered_set_of<z3::ast, z3::ast_ptr_hash, z3::ast_ptr_equal_to>, mpl_::na, mpl_::na, mpl_::na> > >(boost::intrusive_ptr<expr::ENode>, z3::context&, boost::bimaps::views::unordered_map_view<boost::bimaps::relation::member_at::left, boost::bimaps::detail::bimap_core<boost::bimaps::unordered_set_of<boost::intrusive_ptr<expr::ENode>, boost::hash<boost::intrusive_ptr<expr::ENode> >, std::equal_to<boost::intrusive_ptr<expr::ENode> > >, boost::bimaps::unordered_set_of<z3::ast, z3::ast_ptr_hash, z3::ast_ptr_equal_to>, mpl_::na, mpl_::na, mpl_::na> >&, std::unordered_map<boost::intrusive_ptr<expr::ENode>, z3::ast, std::hash<boost::intrusive_ptr<expr::ENode> >, std::equal_to<boost::intrusive_ptr<expr::ENode> >, std::allocator<std::pair<boost::intrusive_ptr<expr::ENode> const, z3::ast> > >&) (./run/bin/seahorn+0xd4f3e1)
#6 0x0000000000d4d8f8 z3::ast ufo::BasicExprMarshal<ufo::FailMarshal>::marshal<boost::bimaps::views::unordered_map_view<boost::bimaps::relation::member_at::left, boost::bimaps::detail::bimap_core<boost::bimaps::unordered_set_of<boost::intrusive_ptr<expr::ENode>, boost::hash<boost::intrusive_ptr<expr::ENode> >, std::equal_to<boost::intrusive_ptr<expr::ENode> > >, boost::bimaps::unordered_set_of<z3::ast, z3::ast_ptr_hash, z3::ast_ptr_equal_to>, mpl_::na, mpl_::na, mpl_::na> > >(boost::intrusive_ptr<expr::ENode>, z3::context&, boost::bimaps::views::unordered_map_view<boost::bimaps::relation::member_at::left, boost::bimaps::detail::bimap_core<boost::bimaps::unordered_set_of<boost::intrusive_ptr<expr::ENode>, boost::hash<boost::intrusive_ptr<expr::ENode> >, std::equal_to<boost::intrusive_ptr<expr::ENode> > >, boost::bimaps::unordered_set_of<z3::ast, z3::ast_ptr_hash, z3::ast_ptr_equal_to>, mpl_::na, mpl_::na, mpl_::na> >&, std::unordered_map<boost::intrusive_ptr<expr::ENode>, z3::ast, std::hash<boost::intrusive_ptr<expr::ENode> >, std::equal_to<boost::intrusive_ptr<expr::ENode> >, std::allocator<std::pair<boost::intrusive_ptr<expr::ENode> const, z3::ast> > >&) (./run/bin/seahorn+0xd4d8f8)
#7 0x0000000000d4d8f8 z3::ast ufo::BasicExprMarshal<ufo::FailMarshal>::marshal<boost::bimaps::views::unordered_map_view<boost::bimaps::relation::member_at::left, boost::bimaps::detail::bimap_core<boost::bimaps::unordered_set_of<boost::intrusive_ptr<expr::ENode>, boost::hash<boost::intrusive_ptr<expr::ENode> >, std::equal_to<boost::intrusive_ptr<expr::ENode> > >, boost::bimaps::unordered_set_of<z3::ast, z3::ast_ptr_hash, z3::ast_ptr_equal_to>, mpl_::na, mpl_::na, mpl_::na> > >(boost::intrusive_ptr<expr::ENode>, z3::context&, boost::bimaps::views::unordered_map_view<boost::bimaps::relation::member_at::left, boost::bimaps::detail::bimap_core<boost::bimaps::unordered_set_of<boost::intrusive_ptr<expr::ENode>, boost::hash<boost::intrusive_ptr<expr::ENode> >, std::equal_to<boost::intrusive_ptr<expr::ENode> > >, boost::bimaps::unordered_set_of<z3::ast, z3::ast_ptr_hash, z3::ast_ptr_equal_to>, mpl_::na, mpl_::na, mpl_::na> >&, std::unordered_map<boost::intrusive_ptr<expr::ENode>, z3::ast, std::hash<boost::intrusive_ptr<expr::ENode> >, std::equal_to<boost::intrusive_ptr<expr::ENode> >, std::allocator<std::pair<boost::intrusive_ptr<expr::ENode> const, z3::ast> > >&) (./run/bin/seahorn+0xd4d8f8)
#8 0x0000000000d4a830 ufo::ZContext<ufo::BasicExprMarshal<ufo::FailMarshal>, ufo::BasicExprUnmarshal<ufo::FailUnmarshal> >::toAst(boost::intrusive_ptr<expr::ENode>) (./run/bin/seahorn+0xd4a830)
#9 0x0000000000d48222 ufo::ZSolver<ufo::ZContext<ufo::BasicExprMarshal<ufo::FailMarshal>, ufo::BasicExprUnmarshal<ufo::FailUnmarshal> > >::assertExpr(boost::intrusive_ptr<expr::ENode>) (./run/bin/seahorn+0xd48222)
#10 0x0000000000dfed13 seahorn::BmcEngine::encode(bool) (./run/bin/seahorn+0xdfed13)
#11 0x0000000000e1124f (anonymous namespace)::BmcPass::runOnModule(llvm::Module&) (./run/bin/seahorn+0xe1124f)
#12 0x00000000009adf28 llvm::legacy::PassManagerImpl::run(llvm::Module&) (./run/bin/seahorn+0x9adf28)
#13 0x00000000006d424f main (./run/bin/seahorn+0x6d424f)
#14 0x00007fa3e5981b97 __libc_start_main (/lib/x86_64-linux-gnu/libc.so.6+0x21b97)
#15 0x00000000006d2eba _start (./run/bin/seahorn+0x6d2eba)
Stack dump:
0.      Program arguments: ./run/bin/seahorn --horn-bmc-engine=mono --horn-sea-dsa=true --horn-bmc --horn-bv2=true --log=opsem --keep-shadows=true --horn-solve test-globals/gint.ll
1.      Running pass 'BmcPass' on module 'test-globals/gint.ll'.
Segmentation fault (core dumped)
```